### ISSUE TITLE
Please update your installation documentation

### ISSUE THREAD

**Issue by awtur** [ipsingh06/seedsync#89]
Hi, 

I installed Docker for Windows, confirmed that it's running, and ran your code from this page: https://ipsingh06.github.io/seedsync/install/
But it appears that the code is either incorrect or obsolete. Can you please double check and update?

I get unrecognized command errors in CMD when I type this: 
`docker run \
   -p 8800:8800 \
   -v <downloads directory>:/downloads \
   -v <config directory>:/config \
   ipsingh06/seedsync`

(Note: the directories were specified)

And this:
`docker-machine ip`

thanks

**Comment by quadcom** [comment]
Just asking,

Dude you change <downloads directory> with your actual local download directory? Also <config directory>?

Those are mapping fields. The dir "/downloads" inside the Docker container will be mapped to the local directory you define. Same goes for config.

**Comment by quadcom** [comment]
Can you provide the exact Docker run command you entered?

**Comment by awtur** [comment]
I realized after starting this thread that the code format was not copy-paste ready and I had to remove all of the  dashes (\\). I'm sure this has thrown off many users who are not used to programming, so I recommend to the author to provide the code without those breaks, or at least to instruct users to remove them.

And second, when I re-run the code, I get an error saying the config is incomplete. Here's the output:
```
C:\Users\xxx>docker run -p 8800:8800 -v C:\Users\xxx\Downloads:/downloads -v "C:\xxx files\install files\Docker_config":/config ipsingh06/seedsync
2021-02-16 05:13:03,881 - INFO - seedsync (MainProcess/MainThread) - Debug mode is disabled.
2021-02-16 05:13:03,888 - INFO - seedsync (MainProcess/MainThread) - Starting SeedSync
2021-02-16 05:13:03,890 - INFO - seedsync (MainProcess/MainThread) - Platform: x86_64
2021-02-16 05:13:04,550 - INFO - seedsync.WebApp (MainProcess/MainThread) - Html path set to: /app/html
2021-02-16 05:13:04,559 - ERROR - seedsync (MainProcess/MainThread) - Config is incomplete
```

And from what I've read, the docker-machine ip command is obsolete.

**Comment by quadcom** [comment]
It looks like you're missing a \ in your config mapping unless "xxx files" is correct?

I'm wondering if the spaces may be causing issues. If I recall, Linux systems don't like spaces in file names or directories. 


Edit: I did a quick search and you're not alone. There are a number of threads about mapping volumes with spaces and people having problems. I bet if you simplify the path structure it'll work.

**Comment by awtur** [comment]
No, the directories are specified correctly; I just replaced my name with 'xxx'. If you have spaces without quotes, such as C:\docker config:/config instead of "C:\docker config":/config, it will of course not run any of the code, but as I posted earlier, it does run the code, only it says the configuration is incomplete. 

I can also confirm that docker will create your specified directories if they don't exist already-- or at the very least the config directory. 

Here are the contents of the configuration file that is created:
```
[General]
debug = False
verbose = False

[Lftp]
remote_address = <replace me>
remote_username = <replace me>
remote_password = <replace me>
remote_port = 22
remote_path = <replace me>
local_path = <replace me>
remote_path_to_scan_script = /tmp
use_ssh_key = False
num_max_parallel_downloads = 2
num_max_parallel_files_per_download = 4
num_max_connections_per_root_file = 4
num_max_connections_per_dir_file = 4
num_max_total_connections = 16
use_temp_file = False

[Controller]
interval_ms_remote_scan = 30000
interval_ms_local_scan = 10000
interval_ms_downloading_scan = 1000
extract_path = /tmp
use_local_path_as_extract_path = True

[Web]
port = 8800

[AutoQueue]
enabled = True
patterns_only = False
auto_extract = True
```

Perhaps someone can point out what's missing. 

Then, there's the issue with finding the IP, as "docker-machine ip" is obsolete. I tried inspecting the entire contents of the container settings and there was no IP there.

**Comment by quadcom** [comment]
Ok, have your pulled up seedsync yet? Can you?

To me it looks like the config just needs your seedbox details.

**Comment by awtur** [comment]
Yeah, I finally got it to work. All I had to do is go to localhost:8800 and fill out the seedbox details.

It's unfortunate that the instructions don't make that clear. They imply that you need to first find the docker/container ip. 
`Open <ip address>:8800 in your browser. In this example that would be http://192.168.100.17:8800`

I would suggest to the author to please clean up the instructions, because if I wasted hours trying to figure out what's wrong, I'm sure that there are plenty of far less experienced users who wouldn't have gotten this far. 

And, what's worse, for me, is that this application does not do what I thought it would. 😩 Somebody on Reddit suggested it for transferring files between two seedboxes (can't find the link anymore), whereas this is clearly remote-->local transfers only. I use FileZilla for that and get the same speeds.

Anyways, thanks for your help @quadcom.
